src/librustc/middle/lang_items.rs

```diff
@@ -90,6 +90,10 @@ impl LanguageItems {
         }
     }
 
+    pub fn require_owned_box(&self) -> Result<ast::DefId, String> {
```

Comment:
My reasoning was:
* owned_box is already somewhat special - it doesn't follow the idioms of the others
* we could have done that but didn't, which indicates that we need to do something differently
* the 'right' thing should be at least as ergonomic as the 'wrong' thing (`.owned_box()`)